.github/workflows/simulator-nightly.yml

```
@@ -60,10 +60,20 @@ jobs:
 
     steps:
     - uses: technote-space/workflow-conclusion-action@v3
+
+    - name: Checkout sui repo main branch
+      uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # pin@v3
+    
+    - name: Get sui commit
+      run: |
+        export sui_sha=$(git rev-parse HEAD)
+        echo "sui_sha=${sui_sha}" >> $GITHUB_ENV
```

Does casing matter here?

```suggestion
        echo "SUI_SHA=${sui_sha}" >> $GITHUB_ENV
```

---

In this case, it does. Line 76 refers to it.